plover/command/set_config.py

```diff
@@ -0,0 +1,29 @@
+from plover import log
+
+
+def set_config(engine, cmdline):
+    """
+    Set a Plover config option to a given value upon executing a stroke pattern.
+
+    Example usage (enter these lines into dictionary):
+    "O*EP": "{PLOVER:SET_CONFIG:translation_frame_opacity,100}",
+    "TR*PB": "{PLOVER:SET_CONFIG:translation_frame_opacity,0}",
+    """
+    # Break down the command string into a key and a value. Throw an error if it isn't in the form of a 2-tuple.
+    try:
+        key, value = tuple(cmdline.split(','))
```

Comment:
What if the value itself contains a comma? Might be better more consistent to split on `:` and only once:
```python
         key, value = tuple(cmdline.split(':', 1))
```

Reply:
The value could just as easily contain a colon, but splitting just once does the trick no matter what the delimiter is (as long as the key can't contain it). Will do.

(I just thought a comma looked better in the dictionary. "SET_CONFIG:key,value" is easier to parse visually than "SET_CONFIG:key:value" for me. This one just comes down to preference, I guess.)